step-plans/step-plans-simple-yaml-parser/src/main/java/step/plans/simple/YamlPlanFields.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (C) 2020, exense GmbH
+ *
+ * This file is part of STEP
+ *
+ * STEP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * STEP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with STEP.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package step.plans.simple;
+
+public class YamlPlanFields {
+    public static final String CALL_FUNCTION_ARGUMENT_ORIGINAL_FIELD = "argument";
+    public static final String CALL_FUNCTION_ARGUMENT_SIMPLE_FIELD = "inputs";
+
+    public static final String CALL_FUNCTION_FUNCTION_ORIGINAL_FIELD = "function";
+    public static final String CALL_FUNCTION_FUNCTION_SIMPLE_FIELD = "keyword";
+
+    public static final String TOKEN_SELECTOR_TOKEN_ORIGINAL_FIELD = "token";
+    public static final String TOKEN_SELECTOR_TOKEN_SIMPLE_FIELD = "selectionCriteria";
```

Comment:
@jeromecomte 
I've renamed the 'selectionCriteria' to 'routing'. For now we have the following ways to define the 'CallKeyword' in yaml (listed in test-selection-criteria-plan.yml).

1. Call keyword by constant name without routing
```
        CallKeyword:
          nodeName: CallKeyword
          keyword: "myKeyword"
```
          
2. Call keyword by dynamic name without routing
```
      CallKeyword:
          nodeName: CallKeyword2
          keyword:
            expression: "'myKeyword2'"
```

3. Call keyword by constant name with routing
```
        CallKeyword:
          nodeName: CallKeyword4
          keyword:
            name:
              expression: "'myKeyword4'"
            routing:
              - criteria1: "criteriaValue1"
```

4. Call keyword by dynamic name with routing (and with inputs)
```
        CallKeyword:
          nodeName: CallKeyword4
          keyword:
            name:
              expression: "'myKeyword4'"
            routing:
              - criteria1: "criteriaValue1"
          inputs:
            - stringInput:
                expression: "'abc'"
            - intInput: 777
```

Reply:
@iegorov777 
- "routing" should be a sibling of "keyword" and "inputs"
- "nodeName" this shouldn't be mandatory. If not specified this should behave like in the visual plan editor and use the keyword name as nodeName
- all fields except the control names are in camelCase with the first letter in lower case. Would it be possible to accept camelCase for the control names too

Reply:
@jeromecomte 
Finally I've changed all control names to lowerCamelCase in yaml format. I've also changed samples in hugo doc and in sample repository.

Reply:
@jeromecomte 
Regarding the "nodeName" I've fixed the behavior. Now for 'CallKeyword' the keyword name (if not dynamic) is used as default "nodeName". Otherwise we use the operator name as default value.